sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -229,8 +229,18 @@ pi_result piDeviceGetInfo(pi_device device, pi_device_info paramName,
     std::memcpy(paramValue, &result, sizeof(cl_bool));
     return PI_SUCCESS;
   }
-  case PI_EXT_ONEAPI_DEVICE_INFO_BFLOAT16:
-    return PI_ERROR_INVALID_VALUE;
+  case PI_EXT_ONEAPI_DEVICE_INFO_BFLOAT16: {
+    size_t extSize;
```

Comment:
Is bfloat16 not always now implemented in hardware or emulated in opencl like for level_zero?

Reply:
bfloat16 conversions are done using hardware when evailable, emulation otherwise.